<html>
<head>
    <meta charset="UTF-8">
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta name="renderer" content="webkit">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Design</title>
    <style type='text/css'>
        #top_title, #sideMenu {
            display: none;
        }

        #content {
            width: 100%;
            margin: 0;
            padding: 0;
        }
    </style>
    <!--jQuery-->
    <script src="http://2016.igem.org/Template:UCAS/jQuery/jquery_min?action=raw&ctype=text/javascript"></script>
    <script src="http://2016.igem.org/Template:UCAS/jQuery/jquery_easing?action=raw&ctype=text/javascript"></script>
    <!--bootstrap-->
    <link href="http://2016.igem.org/Template:UCAS/bootstrap/css/bootstrap_min?action=raw&ctype=text/css" rel="stylesheet">
    <script src="http://2016.igem.org/Template:UCAS/bootstrap/js/bootstrap_min?action=raw&ctype=text/javascript"></script>
    <!--ZalkiHoverImg-->
    <link href="http://2016.igem.org/Template:UCAS/ZalkiHoverImg/zalki_hover_img?action=raw&ctype=text/css" rel="stylesheet">
    <script type="text/javascript" src="http://2016.igem.org/Template:UCAS/ZalkiHoverImg/zalki_hover_img_min?action=raw&ctype=text/javascript"></script>
    <!--customer-->
    <script src="http://2016.igem.org/Template:UCAS/customer/docs_js?action=raw&ctype=text/javascript"></script>
    <link href="http://2016.igem.org/Template:UCAS/customer/docs?action=raw&ctype=text/css" rel="stylesheet">
    <link href="http://2016.igem.org/Template:UCAS/customer/patch?action=raw&ctype=text/css" rel="stylesheet">
    <link href="http://2016.igem.org/Template:UCAS/nav/nav?action=raw&ctype=text/css" rel="stylesheet">
    <link href="http://2016.igem.org/Template:UCAS/customer/customer?action=raw&ctype=text/css" rel="stylesheet">

    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
    <script src="http://cdn.bootcss.com/html5shiv/3.7.0/html5shiv.js"></script>
    <script src="http://cdn.bootcss.com/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->
    <script>
        $(function () {
            $('#top_menu_under').height(6);
        });
    </script>
</head>
<body>
    <div class="container-fluid" style="background-color: black" id="top">
        <img src="http://2016.igem.org/wiki/images/d/dd/T--UCAS--navLeft.png" style="float: left; height: 40px; padding-left: 10px">
        <img src="http://2016.igem.org/wiki/images/2/22/T--UCAS--navRight.png" style="float: right; height: 40px; padding-right: 10px">
        <nav class="homeNav" style="float: right">
            <ul>
                <li><a href="http://2016.igem.org/Team:UCAS">HOME</a></li>
                <li>
                    <a>TEAM</a>
                    <ul>
                        <li><a href="http://2016.igem.org/Team:UCAS/Team">Team</a></li>
                        <li><a href="http://2016.igem.org/Team:UCAS/Collaborations">Collaborations</a></li>
                    </ul>
                </li>
                <li>
                    <a>PROJECT</a>
                    <ul>
                        <li><a href="http://2016.igem.org/Team:UCAS/Description">Description</a></li>
                        <li><a href="http://2016.igem.org/Team:UCAS/Design">Design</a></li>
                        <li><a href="http://2016.igem.org/Team:UCAS/Experiments">Protocols</a></li>
                        <li><a href="http://2016.igem.org/Team:UCAS/Proof">Proof of Concept</a></li>
                        <li><a href="http://2016.igem.org/Team:UCAS/Demonstrate">Demonstrate</a></li>
                        <li><a href="http://2016.igem.org/Team:UCAS/Results">Achievements</a></li>
                        <li><a href="http://2016.igem.org/Team:UCAS/Hardware">Hardware</a></li>
                        <li><a href="http://2016.igem.org/Team:UCAS/Notebook">Notebook</a></li>
                    </ul>
                </li>
                <li>
                    <a>Parts</a>
                    <ul>
                        <li><a href="http://2016.igem.org/Team:UCAS/Parts">Parts</a></li>
                        <li><a href="http://2016.igem.org/Team:UCAS/Basic_Part">Basic Parts</a></li>
                        <li><a href="http://2016.igem.org/Team:UCAS/Composite_Part">Composite Parts</a></li>
                        <li><a href="http://2016.igem.org/Team:UCAS/Part_Collection">Part Collection</a></li>
                    </ul>
                </li>
                <li>
                    <a href="http://2016.igem.org/Team:UCAS/Safety">Safety</a>
                </li>
                <li><a href="http://2016.igem.org/Team:UCAS/Model">Model</a></li>
                <li>
                    <a>HUMAN PRACTICES</a>
                    <ul>
                        <li><a href="http://2016.igem.org/Team:UCAS/Human_Practices">Human Practices</a></li>
                        <li><a href="http://2016.igem.org/Team:UCAS/HP/Silver">Silver</a></li>
                        <li><a href="http://2016.igem.org/Team:UCAS/HP/Gold">Gold</a></li>
                    </ul>
                </li>
                <li>
                    <a href="http://2016.igem.org/Team:UCAS/Attributions">Attributions</a>
                </li>
            </ul>
        </nav>
    </div>
    <div class="container home-content bs-docs-container">
        <div class="row clearfix">
            <div class="col-xs-9 col-sm-9 col-md-9 mid_col">
                <div class="bs-docs-section">
                    <h1 id="js-data-design1" class="page-header">Introduction</h1>
                    <p style="font-size:18px;">
                        There are mainly three concerns in our project design. First of all, degrading enzyme is undoubtedly the core of our project. Whilst natural existing antibiotic resistant enzymes serve as a pool for finding the right ones, other enzymes with similar mechanisms also have the potential to degrade antibiotics at a high rate. For example, a number of resistant proteins degrade antibiotics by oxidizing them, so other oxidases with similar or same coenzymes are highly likely to degrade antibiotics in similar ways. This is a key principle when we were choosing enzymes to test. However, the real situation is much more complex than that. Since many of the oxidases require coenzymes or cofactors to function, they may have problems expressing in prokaryotic cells or for in vitro characterization. This is also a problem we need to tackle.
                    </p>
                    <p style="font-size:18px;">
                        When designing a synthetic biological system that is aimed to function outside laboratories, safety always be the priority. Genetically modified bacteria mainly pose threats to the environment through two ways: horizontal gene transfer (HGT) and bacteria escape.
                    </p>
                    <p style="font-size:18px;">
                        Currently there are mainly three strategies to design a ‘kill-switch’: (1) Suicide system, where toxins or essentials genes are commonly used to mediate cell death
                        <sup>[1],[2]</sup>
                        ;(2) Codon reassignment methods change the corresponding amino acid of a certain codon
                        <sup>[3]</sup>
                        , like stop codon, and sometimes unnatural amino acids are assigned to that ‘new’ codon
                        <sup>[4]</sup>
                        ; (3) Auxotroph is also commonly used. These bacteria depend on an artificial molecule added to the working environment to survive. If bacteria escape to places where that molecule is not found, they will not be able to grow normally.
                    </p>
                    <p style="font-size:18px">
                        No matter what method is taken to build the ‘kill-switch’, a good one should meet some criteria, namely, to prevent mutagenic drift
                        <sup>[3]</sup>
                        , environment supplement and HGT. Mutagenic drift is particularly important for toxic proteins- or essential genes-based ‘kill switches’, for the selective pressure imposed on bacteria may lead to a dramatic alteration in the gene frequency. For those ‘kill-switches’ which make use of auxotrophies, environment supplement is a major challenge. To be more specific, auxotrophic genetically modified microorganisms (GMOs) rely on a kind of substance added into the working environment. However, similar substances may naturally exist in environment, or in an alternative way, bacteria take essential nutrient from remaining of decayed cells.
                    </p>
                </div>
                <div class="bs-docs-section">
                    <h1 id="js-data-design2-0" class="page-header">Design</h1>
                    <h2 id="js-data-design2-1">Part I: Degrading Enzymes</h2>
                    <p style="font-size:18px;">
                        After literature survey, we eventually chose four kinds of oxidase which can be expressed in E.coli and hopefully have opportunities to degrade tetracycline: MnCcP, CPO；FtmOx1, tet X.
                    </p>
                    <p style="font-size:18px;">
                        CPO: catalytic oxidative degradation of sulfamethoxazole by CPO-H2O2 in the presence of chloridion CPO is considered to be the most versatile heme-containing enzyme, exhibiting peroxidase, catalase and cytochrome P450-like activities
                        <sup>[5]</sup>
                        .
                    </p>
                    <p style="font-size:18px;">
                        MnCcP : Crude MnP produced by Phanerochaete chrysosporium was used to degrade tetracycline (TC) and oxytetracycline (OTC) in the presence of Mn2+ and H2O2
                        <sup>[6]</sup>
                        .However, it will form an inclusion body when expressed in prokaryotic organism so we find a designed protein. MnCcP is a designed CcP with a Mn(II)-binding site by making Gly41Glu, Val45Glu, and His181Asp mutations, which can be expressed in procaryotic organisms.
                        <sup>[7]</sup>
                        .
                    </p>
                    <p style="font-size:18px;">
                        FtmOx1: Fumitremorgin B endoperoxidase, is an α-ketoglutarate (α-KG)-dependent mononuclear non-haem iron enzyme. molecular oxygen (O2) is incorporated into verruculogen without O–O bond scission, which lead to the oxidative transformations of substrate this Mononuclear non-haem iron enzymes catalyse a wide range of reactions
                        <sup>[8]</sup>
                        .
                    </p>
                    <p style="font-size:18px;">
                        Tet X:TetX monooxygenase catalyzes regioselective hydroxylation at carbon 11a of tetracyclines. In solutions of pH greater than 1, the product 11a-hydroxytetracycline can decomposes rapidly and non-enzymatically into products that are not easily identifiable
                        <sup>[9]</sup>.
                    </p>
                    <div>
                        <img style="width: 40%" src="http://2016.igem.org/wiki/images/7/76/T--UCAS--design_Fig1.png" />
                        <br />
                        <strong style="text-align: center;color: red;">Fig. 1</strong>
                        <br />
                    </div>
                    <p style="font-size:18px;">
                        The monooxygenase reaction mechanism relies on the redox
                        properties of FAD. After reduction to FADH2 by NADPH, the isoalloxazine binds molecular oxygen to form a hydroperoxide. FAD hydroperoxide is formed after substrate recognition, which subsequently direct substrate hydroxylation takes place
                        <sup>[10]</sup>
                        .
                    </p>
                    <div>
                        <img style="width: 40%" src="http://2016.igem.org/wiki/images/7/76/T--UCAS--design_fig2.png" />
                        <br />
                        <strong style="text-align: center;color: red;">Fig. 2</strong>
                        <br />
                    </div>
                    <p style="font-size:12px;width: 30%;font-style:oblique;font-weight:bold;">
                        Ribbon plot of TetX monooxygenase indicating FAD (stick model) and substrate binding site.
                    </p>
                    <p style="font-size:18px;">
                        After a series of primary lab experiments such as transformation, protein expression, purification of protein, enzyme assay and so on, we intended to accomplish the screen tests of all kinds of enzymes we have selected, as well as enzyme assay tests and kinetics tests for enzymes with high tetracycline degradation efficiency. Enzyme with the best degradation performance would stand out through our screen and would be applied to the construction of engineered bacteria, whose degradation effect in vivo would be tested in further experiments. The plasmid containing MnCcP and FtmOx1 sequences are provided by Prof. Jiangyun Wang’s Lab at Institute of Biophysics, Chinese Academy of Sciences, while the other two enzymes gene sequence are commercially synthesized.
                    </p>
                    <p style="font-size:18px;">
                        In our experiment design, we just took tetracycline degradation as an proof-of-concept. Actually our goal is to apply our engineered bacteria to the degradation of as many as possible kinds of antibiotics which was theoretically feasible with different degradation enzyme and our system (details are shown below in “Extension”).
                    </p>
                </div>
                <div class="bs-docs-section">
                    <h2 id="js-data-design2-2">Part II: Circuit</h2>
                    <h3 id="js-data-design2-2-1">Enhance the expression level of enzymes</h3>
                    <p style="font-size:18px;">
                        In order to transform an ordinary E.coli into a terrific scavenger who can eliminate tetracycline(Tc), tetX gene has to be imported to E.coli, enabling it to degrade Tc. In the beginning, we put tetX gene directly under the control of pTet, a tetracycline-inducible promoter, so that the expression level of tetX will rise as the concentration of Tc goes up.
                    </p>
                    <div>
                        <img style="width: 90%" src="http://2016.igem.org/wiki/images/f/f3/T--UCAS--design_fig3.png" />
                        <br />
                        <strong style="text-align: center;color: red;">Fig. 3</strong>
                        <br />
                    </div>
                    <p style="font-size:12px;width: 80%;font-style:oblique;font-weight:bold;">
                        Design for scavenger. This figure also illustrates our ‘kill-switch’ design, see below.
                    </p>
                    
                    <p style="font-size:18px;">
                        However, this scavenger may not work as hard as we have expected. To further improve the performance of the scavenger, T7 RNA polymerase (T7 RNAP) and T7 promoter are introduced to raise the expression level of tetX. In the presence of T7 RNAP, T7 promoter is recognized and activated, leading to increased transcription of downstream gene. Due to this high activity of T7 promoter, tetX protein can be produced in large quantity when tetX gene is expressed from T7 promoter. The gene coding T7 RNAP is placed downstream of pTet so that when the concentration of tetracycline increases, the amount of T7 RNAP will also increase to produce more tetX.
                    </p>
                    <div>
                        <img style="width: 90%" src="http://2016.igem.org/wiki/images/f/fd/T--UCAS--design_fig4.png" />
                        <br />
                        <strong style="text-align: center;color: red;">Fig. 4</strong>
                        <br />
                    </div>
                    <p style="font-size:12px;width: 80%;font-style:oblique;font-weight:bold;">
                        Design for captain scavenger. This figure also illustrates our ‘kill-switch’ design, see below.
                    </p>
                    <p style="font-size:18px;">
                        We expected the circuit with T7 RNAP to perform better than the one without. For this reason, the E.coli bearing the original circuit is called ‘Scavenger’, and the E.coli bearing the circuit with T7 RNAP is called ‘Captain Scavenger’. 
                    </p>
                    <p style="font-size:18px;">
                        To check the state of the scavengers, a fusion protein tetX-GFP (BBa_K2150013) combining tetX (BBa_K2150101) with GFP (BBa_E0040) is constructed to replace the role of tetX. We can easily determine whether the Scavengers are working by detecting the green fluorescence emitted by them. 
                    </p>
                    <h3 id="js-data-design2-2-2">‘Kill-switch’</h3>
                    <p style="font-size:18px;">
                        In our project, the engineered bacteria carry TetX, a natural antibiotic resistant gene. So it is definitely necessary to incorporate a safe lock to our project.
                    </p>
                    <p style="font-size:18px;">
                        We designed a ‘kill-switch’ based on TA module. TA modules are composed of a toxin molecule and an antitoxin molecule, and can be categorized into five types
                        <sup>[10]</sup>
                        . The antitoxins of both Type I and Type III TA Modules are sRNAs, whereas that of the other three types are small proteins. Among them, Type II TA modules are the most extensively studied, so we decided to choose some of them to build and test our ‘kill-switch’.
                    </p>
                    <p style="font-size:18px;">
                        In order to build a good ‘kill-switch’, there are some major aspects we should consider. First of all, toxins and antitoxins should be readily expressed in E. coli, which is a basic requirement for genetic engineering in all circumstances. This may entail consideration of codon preference and expression levels. Secondly, the toxin must be toxic enough to kill the bacteria with a relatively low dose. However, this does not mean that the toxin is not controllable. It ought to be neutralized by its corresponding protein when, like in the waste water treatment plants (WWTPs), it is not expected to suppress the growth of bacteria.
                    </p>
                    <p style="font-size:20px;">
                        How does it work?
                    </p>
                    <p style="font-size:18px;">
                        As mentioned before, antibiotic residues are everywhere, but the concentration of antibiotics like tetracycline in waste water treatment plants is approximately a thousand times higher than that in nature, such as surface water. This fact is fundamental to our ‘kill-switch’ design. We use the concentration of Tet to control the behavior of bacteria. While staying in the biochemical pool at WWTPs, antitoxin is induced by high concentration of Tet. The large amount of antitoxin binds with toxin, thus neutralizing the toxicity of toxin. In this condition, TetX is also expressed and it oxidases Tet in vivo. The concentration of Tet remains high in the biochemical pools before water leaves it, so bacteria can grow normally in the pool.
                    </p>
                    <p style="font-size:18px;">
                        Nonetheless, if the artificial bacteria escape from the work environment, the concentration of Tet in surroundings drops dramatically. Without sufficient amount of Tet, the transcription of antitoxin is blocked by tetR. In contrast, the expression of toxin will not be impaired in either condition. As antitoxin degrades faster than toxin in the cell, it soon disassociates from toxin, leading to accumulation of free toxins within bacteria cell, which eventually result in cell death or arrest of growth.
                    </p>
                </div>

                <div class="bs-docs-section">
                    <h2 id="js-data-proof7" class="page-header">References</h2>
                    <p style="font-size:12px;font-weight:bold;">
                        [1] Bej, A. K., Perlin, M. H. & Atlas, R. M. Model Suicide Vector for Containment of Genetically Engineered Microorganisms. 54, 2472–2477 (1988).
                    </p>
                    <p style="font-size:12px;font-weight:bold;">
                        Recorbet, G. et al. Conditional suicide system of Escherichia coli released into soil that uses the Bacillus subtilis sacB gene. Appl. Environ. Microbiol. 59, 1361–1366 (1993).
                    </p>
                    <p style="font-size:12px;font-weight:bold;">
                        [3] Ravikumar, A. & Liu, C. C. Biocontainment through reengineered genetic codes. ChemBioChem 16, 1149–1151 (2015).
                    </p>
                    <p style="font-size:12px;font-weight:bold;">
                        [4] Mandell, D. J. et al. Biocontainment of genetically modified organisms by synthetic protein design. Nature 518, 55–60 (2015).
                    </p>
                    <p style="font-size:12px;font-weight:bold;">
                        [5] Xiao Zhang, Xiaohong Li, Yucheng Hu, Shuni Li and Quanguo Zhai. Combination of enzymatic degradation by chloroperoxidase with activated sludge treatment to remove sulfamethoxazole: performance, and eco-toxicity assessment. 4888 jctb(2016)
                    </p>
                    <p style="font-size:12px;font-weight:bold;">
                        [6] Xianghua Wen, Yannan Jia, Jiaxi Li. Enzymatic degradation of tetracycline and oxytetracycline by crude manganese peroxidase prepared from Phanerochaete chrysosporium. Journal of Hazardous Materials. 177 (2010) 924–928
                    </p>
                    <p style="font-size:12px;font-weight:bold;">
                        [7] Thomas D. Pfister, Amir Y. Mirarefi, Alan J. Gengenbach, Xuan Zhao,
                        Connor Danstrom Nicole Conatser, Yi-Gui Gao, Howard Robinson, Charles F. Zukoski, Andrew H.-J. Wang, Yi Lu Kinetic and crystallographic studies of a redesigned manganese-binding site in cytochrome c peroxidase. J Biol Inorg Chem. 12 (2007) 126–137
                    </p>
                    <p style="font-size:12px;font-weight:bold;">
                        [8]WupengYan, Heng Song, Fuhang Song, Yisong Guo, Cheng-Hsuan Wu, Ampon Sae Her, Yi Pu, Shu Wang, Nathchar Naowarojna, Andrew Weitz, Micheal P.Hendrich, Catherine E.Costello, Lixin Zhang, Pinghua Liu&Yan Jessie Zhang. Endoperoxide formation by an α-ketoglutarate-dependent mononuclear non-haem iron enzyme. Nature. 527.(2015) 539-543
                    </p>
                    <p style="font-size:12px;font-weight:bold;">
                        [9] Ian F. Moore, Donald W. Hughes, and Gerard D. Wright. Tigecycline Is Modified by the Flavin-Dependent Monooxygenase TetX. Biochemistry.44, 11829-11835 (2005)
                    </p>
                    <p style="font-size:12px;font-weight:bold;">
                        [10] Gesa Volkers, Gottfried J. Palm, Manfred S. Weiss, Gerard D. Wright, Winfried Hinrichs. Structural basis for a new tetracycline resistance mechanism relying on the TetX monooxygenase. FEBS Letters. 585, 1061-1066(2011)
                    </p>
                    <p style="font-size:12px;font-weight:bold;">
                        [11] Unterholzner, S. J., Poppenberger, B. & Rozhon, W. Toxin-antitoxin systems. Bioengineered 5, 0–13 (2014).
                    </p>
                </div>
            </div>





            <div class="col-xs-3 col-sm-3 col-md-3">
                <div class="bs-docs-sidebar hidden-print hidden-xs hidden-sm" role="complementary">
                    <ul class="nav bs-docs-sidenav" style="margin-left: 0px;">
                        <li>
                            <a href="#js-data-design1">Introduction</a>
                        </li>
                        <li>
                            <a href="#js-data-design2-0">Design</a>
                            <ul class="nav">
                                <li>
                                    <a href="#js-data-design2-1">Part 1</a>
                                </li>
                                <li>
                                    <a href="#js-data-design2-2">Part 2: Circuit</a>
                                    <ul class="nav">
                                        <li><a href="#js-data-design2-2-1">Enhance the expression level of enzymes</a></li>
                                        <li><a href="#js-data-design2-2-2">‘Kill-switch’</a></li>
                                    </ul>
                                </li>
                            </ul>
                        </li>
                        <li>
                            <a href="#js-data-proof7">References</a>
                        </li>
                    </ul>
                    <a class="back-to-top" href="#top">
                        Back to top
                    </a>
                </div>
            </div>
        </div>
    </div>
</body>
</html>